src/v2/BaseLoginRouter.js

```diff
@@ -80,6 +80,8 @@ export default Router.extend({
       settings: this.settings,
     });
 
+    this.hasControllerRendered = false;
```

Comment:
we can put this above `initialize`.. as a property.

Reply:
done